AnnoyancesFilter/sections/subscriptions.txt

```diff
@@ -42,6 +42,8 @@ turkuazgazetesi.net,yenialanya.com,mansetaydin.com,seskocaeli.com,kocaelibarisga
 ||static.klaviyo.com/onsite/js/klaviyo.js$domain=stetson.com
 molistar.com##.box-post-share-gg
 molistar.com##.form-register
+colgate.com#$#.fancybox-overlay-fixed { display: none !important; }
+colgate.com#$#.fancybox-lock { overflow: auto !important; }
```

Comment:
```
colgate.com#$?#html.fancybox-lock { overflow: auto !important; }
```
They have `important` in style.

Reply:
`||u939.colgate.com^` also can safely block that popup(add to both filters).